core/string/ustring.cpp

```diff
@@ -3439,20 +3439,29 @@ String String::replacen(const String &p_key, const String &p_with) const {
 }
 
 String String::repeat(int p_count) const {
-	ERR_FAIL_COND_V_MSG(p_count < 0, "", "Parameter count should be a positive number.");
+	ERR_FAIL_COND_V_MSG(p_count < 0, "", "Parameter count should be a non-negative number.");
```

Comment:
I don't see this as much of an improvement. It's better to say what it should be than what it shouldn't be - and 0 is positive too (as well as negative, if you want, but that doesn't matter here).

Reply:
This change was suggested from kleonc's review 😅

Mathematically speaking, I think the consensus is that 0 is neither positive nor negative 🤷

Reply:
To me "positive" is `>= 0`, "strictly positive" is `> 0`.

This isn't a core math method so I don't think deep considerations on what's most mathematically correct are relevant here. And "should be a non-negative number" is really awkward (if anything, it should be "should not be a negative number", but a **positive** worded error is always better.

Reply:
Yeah, to me positive by deafult is `> 0` (unless it's explicitly defined otherwise), strictly positive is just being super clear.
[I've just read](https://fr.wikipedia.org/wiki/Nombre_positif) that in French literature it's quite common that positive means `>= 0`. So maybe it's just a matter of a different background? :thinking: 

Anyway, not gonna bikeshed about it. :smile:

Reply:
Indeed this might be a Frenchism. The English page does mention "non-negative real numbers" for >= 0.
https://en.wikipedia.org/wiki/Positive_real_numbers

Such an awkward name :P